### ISSUE TITLE
[FEATURE]: New option to the Authentication- allow to turn switches on/off 

### ISSUE THREAD

**Issue by Sagi313** [theyosh/TerrariumPI#543]
I turn my lights on/off manually in the "Switches" tab, but every time I wish to do so, I need to enter username and password.

I would be glad is there will be an option (beside "full authentication" and "changes only"), that allows to manage all the switches without authentication.

![image](https://user-images.githubusercontent.com/23437763/114615019-db73b280-9cad-11eb-8bf5-0225d37bd486.png)

Maybe you can consider an option with no authentication at all 🔢 
Thank you

**Comment by theyosh** [comment]
Hi,

you mean that you toggle the relay in the interface? It should remember the login until your close your browser.

I am not a fan of totally disabling authentication. As this can introduce security issues. Especially when people put it online to the internet.  And only making the relay changes without authentication is not an option.

So I could add a 'no authentication' option to it. But I am afraid that this will be done in the new version 4. As I will not make new features in the 3 version. We are already testing version 4.

**Comment by Sagi313** [comment]
> you mean that you toggle the relay in the interface? It should remember the login until your close your browser.
Yes. I close the browser very often, so I have to do a full authentication each time.

I get your point about the security issues, that makes sense, although mine is only on the local network.

anyway, i'll be glad to see this feature in version 4!
Also- Going to upload a picture of my setup this week :)

**Comment by theyosh** [comment]
Hi,

This is now possible in Version 4. If you have a spare Pi and some spare hardware, you could test this...
